dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/StaticVarCompensator/SVarCControl.mo

```diff
@@ -0,0 +1,106 @@
+within Dynawo.Electrical.Controls.Voltage.StaticVarCompensator;
+
+/*
+* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model SVarCControl "Control for standard static var compensator model"
+  import Modelica;
+  import Dynawo.Types;
+  import Dynawo.Connectors;
+  import Dynawo.Electrical.SystemBase;
+  import Dynawo.Electrical.Controls.Voltage.StaticVarCompensator.Parameters;
+
+  extends Parameters.Params_Regulation;
+  extends Parameters.Params_Limitations;
+  extends Parameters.Params_CalculBG;
+  extends Parameters.Params_ModeHandling;
+
+  Modelica.Blocks.Interfaces.RealInput URef(start = URef0) "Voltage reference for the regulation in kV" annotation(
+    Placement(visible = true, transformation(origin = {-168, 26}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, 85}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput UPu(start = U0Pu) "Voltage at the static var compensator terminal in p.u (base UNom)" annotation(
+    Placement(visible = true, transformation(origin = {-168, -2}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, -41}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput QInjPu(start = -Q0Pu) "Reactive power injected by the static var compensator in p.u (base SnRef) (generator convention)" annotation(
+    Placement(visible = true, transformation(origin = {-168, -68}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, -83}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
+
+  Modelica.Blocks.Interfaces.BooleanInput selectModeAuto(start = true) "Mode selected when in manual configuration" annotation(
+    Placement(visible = true, transformation(origin = {-168, 54}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, 49}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.IntegerInput setMode(start = 2) "Wheter the static var compensator is in automatic configuration" annotation(
```

Comment:
Is it normal to have these default start values?

Reply:
it is intended to have default values for these variables but not exactly that way: fixed

added selectModeAuto0 and setModeManual0 with default values that can be overwritten if needed